lib/generation/flutter_project_builder/flutter_project_builder.dart

```diff
@@ -231,8 +243,17 @@ class FlutterProjectBuilder {
                 '${projectName}/lib/main.dart', relPath);
           }
 
-          var page = flutterGenerator.generate(intermediateItem.node);
-
+          var page;
```

Comment:
Is this because we have the assumption that all nodes that have states attached will be detected at the root layer? Just want to make sure that was the intention here because I believe it is correct.

Reply:
That is correct, that was my assumption 👍🏾

Reply:
Awesome. LGTM